R/standard-path.R

```diff
@@ -16,13 +16,13 @@ StandardContainer <- enum(c(
 #' @include utilities-path.R
 #' @export
 StandardPath <- enum(c(
-  Age = toPathString(StandardContainer$Organism, "Age"),
-  Height = toPathString(StandardContainer$Organism, "Height"),
-  Weight = toPathString(StandardContainer$Organism, "Weight"),
-  BMI = toPathString(StandardContainer$Organism, "BMI"),
-  BSA = toPathString(StandardContainer$Organism, "BSA"),
-  GestationalAge = toPathString(StandardContainer$Organism, "Gestational age"),
-  OntogenyFactorAlbumin = toPathString(StandardContainer$Organism, "Ontogeny factor (albumin)"),
+  Age                      = toPathString(StandardContainer$Organism, "Age"),
+  Height                   = toPathString(StandardContainer$Organism, "Height"),
+  Weight                   = toPathString(StandardContainer$Organism, "Weight"),
+  BMI                      = toPathString(StandardContainer$Organism, "BMI"),

```

Comment:
It this done via styler oder by hand?
Styler yes! By hand no

Reply:
Naaaah not doing it! One space all the time!